language-server/modules/langserver-core/src/test/resources/completion/expression_context/source/mapping_constructor_expr_ctx_source4.bal

```diff
@@ -0,0 +1,7 @@
+int moduleVar = 20;
+
+class MyClass {
+    int x = 10;
+    int y = 20;
+    record {int y; int x;} rec = {x, };
```

Comment:
Shall we add another test case here with the record being defined as a type:
```
type Employee record {|
    readonly int empId;
    string name;
|}; 
```
And inside the class:
```
Employee emp = {x, };
```